memeolist/app/src/main/java/org/aerogear/android/app/memeolist/ui/MemeListActivity.java

```diff
@@ -56,6 +57,8 @@
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_meme_list);
+    Login login = new Login();
+    login.createProfile();
```

Comment:
What happens if the profile is already created?

Reply:
It creates new profile every time. I will change that now to check if email exist and create it only then